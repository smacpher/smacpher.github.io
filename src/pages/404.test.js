import NotFound from "./404.js"
import React from "react"
import renderer from "react-test-renderer"

describe("NotFound", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<NotFound />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
