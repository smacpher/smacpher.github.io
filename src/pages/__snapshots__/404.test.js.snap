// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<div
  className="layout"
>
  <div
    className="center"
  >
    <h3>
      404 Not Found.
    </h3>
  </div>
</div>
`;
